### ISSUE TITLE
Entity classes are shadowing the navigation field

### ISSUE THREAD

**Issue by NightrainsRbx** [liopyu/EntityJs#10]
All custom entity classes define a navigation field, shadowing the navigation field of the Mob class, so when it creates the PathNavigation it sets it to that new field instead, since it doesn't override the getNavigation method it calls the one in the Mob class which returns the navigation field of the Mob class.

This makes adding custom PathNavigation not work at all since they are set to the wrong field and never used.

It is necessary in order to make Flying, Aquatic, and Wall Climber entities.

As a feature request, adding a way to change the MoveControl in the entity builder is also necessary, flying entities are required to use FlyingMoveControl.

**Comment by liopyu** [comment]
i see, ill do that as well as add lookControl and jumpControl while im at it

**Comment by liopyu** [comment]
This should now be fixed in the latest EntityJS release [(0.3.6-1.20.1/1.19.2) or (1.1.0-1.21)](https://www.curseforge.com/minecraft/mc-mods/entityjs/files/all?page=1&pageSize=20). I also added methods for creating builders for [LookControlJS ](https://github.com/liopyu/EntityJS/blob/EntityJS-1.20.1-forge/src/main/java/net/liopyu/entityjs/util/ai/LookControlJS.java).  [MoveControlJS ](https://github.com/liopyu/EntityJS/blob/EntityJS-1.20.1-forge/src/main/java/net/liopyu/entityjs/util/ai/MoveControlJS.java) & [JumpControlJS ](https://github.com/liopyu/EntityJS/blob/EntityJS-1.20.1-forge/src/main/java/net/liopyu/entityjs/util/ai/JumpControlJS.java)to the `EntityJSUtils` binding which should allow full customization of these base classes if you decide on going that route.
```javascript
StartupEvents.registry("entity_type", event => {
	event.create("wyrm", "minecraft:axolotl")
		.createNavigation(ctx => {
			return EntityJSUtils.createGroundPathNavigation(ctx.entity, ctx.level)
		})
		.setLookControl(entity => {
			return EntityJSUtils.createLookControl(entity, builder => { })
		})
		.setMoveControl(entity => {
			return EntityJSUtils.createMoveControl(entity, builder => { })
		})
		.setJumpControl(entity => {
			return EntityJSUtils.createJumpControl(entity, builder => { })
		})
})
```

**Comment by NightrainsRbx** [comment]
Yep, that works perfectly, thx!
